content/blog/2024/12/kaoto-release-2.3.0/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Kaoto v2.3 release"
+date: 2024-12-10
+draft: false
+authors: [igarashitm, lordrip]
+categories: ["Releases", "Tooling", "Kaoto"]
+preview: "Kaoto 2.3 has been released"
+---
+
+![Kaoto](cover.png)
+
+Following the Apache Camel 4.9.0 release, we're happy to announce the release of Kaoto 2.3.
+
+## What’s New in Kaoto 2.3?
+
+We're thrilled to announce the release of Kaoto 2.3, bringing new features, improvements, and bug fixes to enhance your integration experience.
+This release also brings the first technical preview of a long awaited feature: the Kaoto DataMapper with the ability to perform data transformations using Camel.
+
+## Here are the key highlights of this release:
```

Comment:
as it is a title, I think I would turn it more in something like:
* Key highlights
* Key highlights of Kaoto 2.3

but on the other had, it seemsvery redundant with the previous "What's new in kaoto 2.3?"